yamlfmt.go

```diff
@@ -101,6 +101,7 @@ type FeatureApplyMode string
 
 var (
 	FeatureApplyBefore FeatureApplyMode = "before"
+	FeatureApplyDuring FeatureApplyMode = "during"
```

Comment:
I think this DuringAction also needs to also be removed.

Reply:
I had done this, but forgot to push 🙃